tests/unit/views/test_dashboard.py

```diff
@@ -1,15 +1,22 @@
 import time
 import unittest
+import sys
 
 from tests.unit import AsyncHTTPTestCase
 from tests.unit.utils import task_succeeded_events, task_failed_events
 from tests.unit.utils import HtmlTableParser
 
+if sys.version_info >= (2, 7):
```

Comment:
`sys.version_info >= (2, 7)` is `False` only for Python 2.6- and those versions are not supported.
Also, I'm quite sure that `from unittest.mock import patch, PropertyMock` desn't work for them anyway.
So you have to remove the try-catch and keep only `from mock import patch, PropertyMock`.